config/database.js

```diff
@@ -0,0 +1,12 @@
+const Sequelize = require('sequelize');
+
+module.exports = new Sequelize(
+  process.env.DATABASE_NAME,
+  process.env.DATABASE_USER,
+  process.env.DATABASE_PASSWORD,
+  {
+    host: process.env.DATABASE_HOST,
+    port: process.env.DATABASE_PORT,
+    dialect: 'mysql',
+  },
+);
```

Comment:
You should add the `sync` thing in here that you've added on the favourites branch. If I downloaded this on my own laptop, I would have to go and create the database tables manually, but if you added `sync` here, it would do that for me.